crates/bevy_render/src/camera/camera.rs

```diff
@@ -935,7 +935,8 @@ pub fn extract_cameras(
                 continue;
             }
 
-            let mut commands = commands.get_or_spawn(entity).insert((
+            let mut commands = commands.entity(render_entity.id());
```

Comment:
Why all the changes to formatting here? Correct me if I'm wrong, but I think there's no real code changes in this file besides the one entity -> render_entity.id(), yes?

Reply:
Yes, that's correct

Reply:
Yes, that's the only substantive change